### PR TITLE
Moving Scroll folders out of experimental

### PR DESCRIPTION
**Pull request by vaoliva**
## Overview

This PR is part of Scroll graduation:

- Moving scripts, folders and example assets linked to ScrollingObjectCollection out of experimental folders.
- Changing namespaces accordingly.

### CODE REVIEW

File: Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Scrolling object collection
-![](../../../../../Documentation/Images/ScrollingCollection/MRTK_UX_ScrollingCollection_Main.jpg)
+![](../../../../../../../Documentation/Images/ScrollingCollection/MRTK_UX_ScrollingCollection_Main.jpg)
```
Comment (wiwei) [inline Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/README.md:2]: This makes me wonder if going forward, we should just put images near where their content - i.e. the journey to the root here is a bit weird.

Anyways, this is a comment out loud just curious about what folks think for future things, not actually suggesting a change for this particular one.

**Review by wiwei — APPROVED** [review]
lgtm modulo the doc build breaks

File: Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/README.md (hunk 1 above)
Comment (vaoliva) [reply Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/README.md:2]: This README is actually going to be deleted. Its the README that comes inside experimental folder in the project for experimental stuff.

Comment (vaoliva) [reply Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/README.md:2]: But for experimental stuff, maybe we should only ask contributors to add READMEs to the actual Documentation folder structure outside the project.

**Pull request merged**